lib/components/viewers/stop-viewer.js

```diff
@@ -203,16 +154,34 @@ class StopViewer extends Component {
    * TODO: Can this use SetFromToButtons?
    */
   _renderControls = () => {
-    const {stopData} = this.props
+    const {homeTimezone, stopData} = this.props
     const {scheduleView} = this.state
+    const userTimeZone = getUserTimezone()
+    const inHomeTimezone = homeTimezone && homeTimezone === userTimeZone
+
     // Rewrite stop ID to not include Agency prefix, if present
     // TODO: make this functionality configurable?
     let stopId
     if (stopData && stopData.id) {
       stopId = stopData.id.includes(':') ? stopData.id.split(':')[1] : stopData.id
     }
+
+    let timezoneWarning
+    if (!inHomeTimezone) {
+      const timezoneCode = moment().tz(homeTimezone).format('z')
+
+      // Display a banner about the departure timezone if user's timezone is not the configured 'homeTimezone'
+      // (e.g. cases where a user in New York looks at a schedule in Los Angeles).
+      timezoneWarning = (
+        <Alert bsStyle='info' style={{ marginBottom: 0, padding: '5px 10px', textAlign: 'center' }}>
```

Comment:
I think this could use a `margin-top: 5px`. Any reason you opted not to use a styled component?

Reply:
No... Fixed in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/08253eff2bb186584000d7209a5635d6de86dbfb.